cognite/client/_api/entity_matching.py

```diff
@@ -0,0 +1,221 @@
+
+        Args:
+            item (Union[EntityMatchingModel,EntityMatchingModelUpdate,List[Union[EntityMatchingModel,EntityMatchingModelUpdate]]) : Model(s) to update
+        """
+        return self._update_multiple(items=item)
+
+    def list(self, filter: Dict = None, limit=100) -> EntityMatchingModelList:
+        """List models
+
+        Args:
+            filter (dict): If not None, return models with parameter values that matches what is specified in the filter.
+            limit (int, optional): Maximum number of items to return. Defaults to 100. Set to -1, float("inf") or None to return all items.
+
+        Returns:
+            EntityMatchingModelList: List of models."""
+        filter = {utils._auxiliary.to_camel_case(k): v for k, v in (filter or {}).items() if v is not None}
+        # NB no pagination support yet
+        models = self._post(self._RESOURCE_PATH + "/list", json={"filter": filter}).json()["items"]
+        return EntityMatchingModelList(
+            [self._LIST_CLASS._RESOURCE._load(model, cognite_client=self._cognite_client) for model in models]
+        )
+
+    def list_jobs(self) -> EntityMatchingModelList:
+        """List jobs
+
+        Returns:
+            EntityMatchingModelList: List of jobs."""
+        return EntityMatchingModelList(
+            [
+                self._LIST_CLASS._RESOURCE._load(model, cognite_client=self._cognite_client)
+                for model in self._get(self._RESOURCE_PATH + "/jobs").json()["items"]
+            ]
+        )
+
+    def delete(self, id: Union[int, List[int]] = None, external_id: Union[str, List[str]] = None) -> None:
+        """Delete models
+
+        Args:
+            id (Union[int, List[int]): Id or list of ids
+            external_id (Union[str, List[str]]): External ID or list of external ids"""
+        self._delete_multiple(ids=id, external_ids=external_id, wrap_ids=True)
+
+    def fit(
+        self,
+        sources: List[Union[Dict, CogniteResource]],
+        targets: List[Union[Dict, CogniteResource]],
+        true_matches: List[Union[Dict, Tuple[Union[int, str], Union[int, str]]]] = None,
+        match_fields: List[Tuple[str, str]] = None,
+        feature_type: str = None,
+        classifier: str = None,
+        ignore_missing_fields: bool = False,
+        name: str = None,
+        description: str = None,
+        external_id: str = None,
+    ) -> EntityMatchingModel:
+        """Fit entity matching model.
+        Note: All users on this CDF subscription with assets read-all capability in the project, are able to access the data sent to this endpoint.
+
+        Args:
+            sources: entities to match from, should have an 'id' field. Tolerant to passing more than is needed or used (e.g. json dump of time series list)
+            targets: entities to match to, should have an 'id' field.  Tolerant to passing more than is needed or used.
+            id_field (str): use 'id' or 'external_id' as the id field to match resources
```

Comment:
what's the diff bw `id_field` and `external_id`?

Reply:
This `id_field` is not in there in the function arugments though.

Reply:
this was an old parameter which we don't have anymore, nice catch